src/screens/market/tezos-info/tezos-info.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Alert } from 'react-native';
+
+import { useTezosMarketCoin } from '../../../store/market/market-selectors';
+import { useColors } from '../../../styles/use-colors';
+import { getPriceChangeColor, getValueToShow, getPriceChange } from '../../../utils/market.util';
+
+export const useTezosInfoService = () => {
```

Comment:
what is the point of this hook? it has no shared logic or logic that could be extracted as a big chunk.